packages/vm/src/runTx.ts

```diff
@@ -205,20 +204,20 @@ async function _runTx(vm: VM, opts: RunTxOpts): Promise<RunTxResult> {
       throw Error(`Verkle access witness needed for execution of verkle blocks`)
     }
 
-    if (
-      !(vm.stateManager instanceof StatefulVerkleStateManager) &&
-      !(vm.stateManager instanceof StatelessVerkleStateManager)
-    ) {
+    if (!('verifyPostState' in vm.stateManager) && !('verifyPostState' in vm.stateManager)) {
```

Comment:
Isn't this checking the same condition twice?

Reply:
Ah, good point, I did not catch this :sweat_smile: If `verifyPostState` is only a method of `StatefulVerkleStateManager` and `StatelessVerkleStateManager` and not of any other StateManager then we can indeed compactify it to one check (and not the same check twice)

Reply:
Hah, sorry I missed that.  Will update!